docs/src/Atmos/Microphysics.md

```diff
@@ -0,0 +1,369 @@
+
+|    symbol            |         definition                                        | units                    | default value         |
+|----------------------|-----------------------------------------------------------|--------------------------|-----------------------|
+|``n_{0_{MP}}``        | rain drop size distribution parameter                     | ``\frac{1}{m^4}``        | ``16 \cdot 10^6``     |
+|``\tau_{cond_evap}``  | cloud water condensation/evaporation timescale            | ``s``                    | ``10``                |
+|``\tau_{acnv}``       | cloud to rain water autoconversion timescale              | ``s``                    | ``10^3``              |
+|``q_{liq\_threshold}``| cloud to rain water autoconversion threshold              | -                        | ``5 \cdot 10^{-4}``   |
+|``E_{col}``           | collision efficiency between rain drops and cloud droplets| -                        | ``0.8``               |
+|``C_{drag}``          | rain drop drag coefficient                                | -                        | ``0.55``              |
+|``a_{vent}, b_{vent}``| rain drop ventilation factor coefficients                 | -                        | ``1.5 \;``,``\; 0.53``|
+|``K_{therm}``         | thermal conductivity of air                               | ``\frac{J}{m \; s \; K}``| ``2.4 \cdot 10^{-2}`` |
+|``\nu_{air}``         | kinematic viscosity of air                                | ``\frac{m^2}{s}``        | ``1.6 \cdot 10^{-5}`` |
+|``D_{vapor}``         | diffusivity of water vapor                                | ``\frac{m^2}{s}``        | ``2.26 \cdot 10^{-5}``|
+
+## Rain drop size distribution
+
+The rain-drop size distribution is assumed to follow Marshall-Palmer distribution (Marshall Palmer 1948 eq. 1):
+```math
+\begin{equation}
+n(r) = n_{0_{MP}} exp\left(- \lambda_{MP} \; r \right)
+\end{equation}
+```
+where:
+ - ``r`` is the drop radius,
+ - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters (twice the values used in the Marshall Palmer 1948, because we use drop radius and not diameter).
+
+## Terminal velocity
+
+The terminal velocity of an individual rain drop is defined by the balance between the gravity (taking into account the density difference between water and air) and the drag force:
+```math
+\begin{equation}
+v_{drop} = \left(\frac{8}{3 \; C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} g^{1/2} \; r^{1/2} = v_c(\rho) \; g^{1/2} \; r^{1/2}
+\end{equation}
+```
+where:
+ - ``g`` is the gravitational acceleration,
+ - ``C_{drag}`` is the drag coefficient,
+ - ``\rho_{water}`` is the density of water,
+ - ``\rho`` is the density of air.
+
+The mass weighted terminal velocity ``v_t`` is defined following Ogura and Takahashi 1971
+```math
+\begin{equation}
+v_t = \frac{F_{rain}}{RWC}
+\end{equation}
+```
+where:
+ - ``F_{rain} = \int_0^\infty n(r) \; m(r) \; v_{drop}(r) dr`` is the vertical flux of rain drops,
```

Comment:
dr -> \, dr

(and I would change \; to \, ...:)

Reply:
done